pkg/backup/backup_planning.go

```diff
@@ -636,7 +636,7 @@ func backupPlanHook(
 			initialDetails.Destination.Subdir = backupbase.LatestFileName
 			initialDetails.Destination.Exists = true
 		} else if subdir != "" {
-			initialDetails.Destination.Subdir = "/" + strings.TrimPrefix(subdir, "/")
+			initialDetails.Destination.Subdir = "/" + strings.TrimSuffix(strings.TrimPrefix(subdir, "/"), "/")
```

Comment:
could you create a follow up issue (GA blocker for 25.4) to validate that every subdir passed to SHOW BACKUP/RESTORE/compaction job also is normalized?